### PR TITLE
Instruction for mill installation on Windows

### PR DESCRIPTION
**Pull request by mazhara**
draft commit

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,9 +46,37 @@ This project requires quite a few tools to get up and running. Don’t worry if
...
 
 You’ll now need to install:
- - [mill version 0.8.0](http://www.lihaoyi.com/mill/)
- - [nodejs version 10](https://nodejs.org/)
- - [yarn version 1.22](https://yarnpkg.com/)
+  
+### Mill
+[Version  0.8.0](http://www.lihaoyi.com/mill/)
+
+####Windows
```
Comment (zainab-ali) [inline CONTRIBUTING.md:53]: There needs to be a space after the # sign here

**Review by zainab-ali — CHANGES_REQUESTED** [review]

**Review by zainab-ali — CHANGES_REQUESTED** [review]
Thanks Olga!
This is a great start.  I've a few suggestions.
While writing the comments, I pushed a few changes to a branch.  Have a look at it for inspiration:
https://github.com/mazhara/lsug-website/compare/windows_inst...zainab-ali:windows_inst

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -81,6 +109,10 @@ Check that the server tests pass.
 mill server.test
 ```
 
+Note: if you are Windows user you might use --no-service optione, like:
+
+`mill server.test --no-service`
```
[the remark this answers is not in the document]
Comment (mazhara) [reply CONTRIBUTING.md]: I have rechecked. mill.bat works. Only difference , for windows command line we can use just 'mill' while for windows PowerShell  './mill'. I have added it 'Checking your setup' remark

[the remark this answers is not in the document]
Comment (zainab-ali) [reply CONTRIBUTING.md]: Thanks!

**Comment by zainab-ali** [comment]
Thanks @mazhara ! This is good to go

**Pull request merged**